lib/author.rb

```diff
@@ -0,0 +1,14 @@
+require 'pry'
+require "./lib/person"
+
+class Author < Person
+  def initialize
```

Comment:
Prob want getters/setters or this class is purpose-less and has no functionality.